pos_product_template/models/pos_session.py

```diff
@@ -0,0 +1,86 @@
+from odoo import api, models
+
+
+class PosSession(models.Model):
+    _inherit = "pos.session"
+
+    def _loader_params_pos_config(self):
+        params = super()._loader_params_pos_config()
+        if params.get("search_params", {}).get("fields"):
```

Comment:
test are not usefull here, as base function return search_params and fields. (otherwise, the PoS will not work at all...)